backend/tests/test_slack_notifier.py

```
@@ -115,12 +125,14 @@ def test_no_http_call_when_not_configured(self, mock_post):
 class TestSlackNotifierFallback:
     """Tests for config fallback behavior."""
 
+    @pytest.mark.unit
     @patch("app.services.slack_notifier.get_slack_webhook_url", return_value="")
     def test_fallback_to_config_empty(self, mock_get_url):
```

Ruff (0.15.13)

[warning] Unused method argument: `mock_get_url`

(ARG002)